### PR TITLE
Body barrier

### PR DESCRIPTION
**Pull request by pierromond**

### CODE REVIEW

File: noisemodelling-jdbc/src/main/java/org/noise_planet/noisemodelling/jdbc/JdbcNoiseMap.java [hunk 1]
```diff
@@ -57,6 +57,16 @@
     // Soil areas are splited by the provided size in order to reduce the propagation time
     protected double groundSurfaceSplitSideLength = 200;
     protected int soundReflectionOrder = 2;
+
+    public boolean isBodyBarrier() {
```
Comment (nicolas-f) [inline noisemodelling-jdbc/src/main/java/org/noise_planet/noisemodelling/jdbc/JdbcNoiseMap.java:61]: missing function description

Comment (pierromond) [reply noisemodelling-jdbc/src/main/java/org/noise_planet/noisemodelling/jdbc/JdbcNoiseMap.java:61]: I never use it, it is only an automatic generation reminiscence...

Comment (pierromond) [reply noisemodelling-jdbc/src/main/java/org/noise_planet/noisemodelling/jdbc/JdbcNoiseMap.java:61]: I need to remove it or you can do ?

Comment (nicolas-f) [reply noisemodelling-jdbc/src/main/java/org/noise_planet/noisemodelling/jdbc/JdbcNoiseMap.java:61]: What ? Clean your own code :P

**Comment by pierromond** [comment]
Done ! 🥇

File: noisemodelling-jdbc/src/main/java/org/noise_planet/noisemodelling/jdbc/JdbcNoiseMap.java [hunk 2]
```diff
@@ -352,6 +344,11 @@ public void fetchCellSource(Connection connection,Envelope fetchEnvelope, Cnosso
         }
     }
 
+
+    public void setBodyBarrier(boolean bodyBarrier) {
```
Comment (nicolas-f) [inline noisemodelling-jdbc/src/main/java/org/noise_planet/noisemodelling/jdbc/JdbcNoiseMap.java:348]: What is body barrier function, default value ? What is the meaning of this boolean ? Please add code documentation

Comment (nicolas-f) [reply noisemodelling-jdbc/src/main/java/org/noise_planet/noisemodelling/jdbc/JdbcNoiseMap.java:351]:
```
2022-03-28T12:16:41.8981959Z Executing test testRailWayEmissionFromDEN [org.noise_planet.noisemodelling.wps.TestNoiseModelling] with result: FAILURE
2022-03-28T12:16:41.8982461Z 
2022-03-28T12:16:41.8982802Z org.noise_planet.noisemodelling.wps.TestNoiseModelling > testRailWayEmissionFromDEN FAILED
2022-03-28T12:16:41.8983568Z     groovy.lang.MissingPropertyException: No such property: DEFAULT_FREQUENCIES_THIRD_OCTAVE for class: org.noise_planet.noisemodelling.propagation.PropagationProcessPathData
2022-03-28T12:16:41.8984335Z         at groovy.lang.MetaClassImpl.invokeStaticMissingProperty(MetaClassImpl.java:1021)
2022-03-28T12:16:41.8984889Z         at groovy.lang.MetaClassImpl.getProperty(MetaClassImpl.java:1866)
2022-03-28T12:16:41.8985385Z         at groovy.lang.MetaClassImpl.getProperty(MetaClassImpl.java:1842)
2022-03-28T12:16:41.8985873Z         at groovy.lang.MetaClassImpl.getProperty(MetaClassImpl.java:3797)
2022-03-28T12:16:41.8986567Z         at org.codehaus.groovy.runtime.callsite.ClassMetaClassGetPropertySite.getProperty(ClassMetaClassGetPropertySite.java:50)
2022-03-28T12:16:41.8987353Z         at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callGetProperty(AbstractCallSite.java:298)
2022-03-28T12:16:41.8988059Z         at org.noise_planet.noisemodelling.wps.NoiseModelling.Railway_Emission_from_Traffic.exec(Railway_Emission_from_Traffic.groovy:154)
2022-03-28T12:16:41.8988723Z         at org.noise_planet.noisemodelling.wps.NoiseModelling.Railway_Emission_from_Traffic$exec.call(Unknown Source)
2022-03-28T12:16:41.8989365Z         at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)
2022-03-28T12:16:41.8989992Z         at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:115)
2022-03-28T12:16:41.8990618Z         at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:135)
2022-03-28T12:16:41.8991331Z         at org.noise_planet.noisemodelling.wps.TestNoiseModelling.testRailWayEmissionFromDEN(TestNoiseModelling.groovy:70)
```

**Pull request merged**